src/formio/validation.ts

```diff
@@ -33,7 +73,9 @@ export type BaseErrorKeys =
   // custom, added by OF
   | 'minTime'
   | 'maxTime'
-  | 'invalid_time';
+  | 'invalid_time'
+  | 'invalid_date'
```

Comment:
this one needs to go in the group above, it is defined and used by formio itself. However, it is used in the context of an invalid *datetime* because the JS `Date` is actually a full datetime.